fdbserver/DataDistribution.actor.cpp

```diff
@@ -3246,7 +3247,11 @@ ACTOR Future<Void> removeWrongStoreType(DDTeamCollection* self) {
 				// Server may be removed due to failure while the wrongStoreTypeToRemove is sent to the
 				// storageServerTracker. This race may cause the server to be removed before react to
 				// wrongStoreTypeToRemove
-				server.second->wrongStoreTypeToRemove.set(true);
+				if (self->configuration.storageMigrationType == StorageMigrationType::AGGRESSIVE) {
```

Comment:
Similar to this, what should the behavior be when migrationType == disabled but the wiggle is on? This would be if you wanted to use the wiggle to just rewrite storage engines.
If someone changed the storage engine, the wiggle would still rewrite it even though migration_type was disabled, right?
It seems like we'd either:
 - want the wiggle to not exclude a server and pause if it had a different type than the desired storage engine configuration
 - not allow the user to change the storage engine type if they have the wiggle enabled and migration type disabled?
 @sfc-gh-mpilman @sfc-gh-xwang thoughts?

Reply:
My personal option is to directly make `configure $new_engine` command failed if migrationType=disabled. Do you feel it more graceful and easier?

Reply:
I like that approach as well. It's simpler, and i prefer preventing weird states instead of handling them.